week39/소수만들기_nahye03.py

```diff
@@ -0,0 +1,39 @@
+#소수 판별 함수
+def is_prime(num):
+    for i in range(2, num):
+        if num % i == 0:
+            return False
+
+    return True
```

Comment:
- 모든 약수는 대칭 형태이기 때문에 num 대신 num의 제곱근까지만 나누어 떨어지는지 확인해 소수 여부를 확인할 수 있습니다.
```suggestion
def is_prime(num):
    for i in range(2, int(num*0.5)+1):
        if num % i == 0:
            return False
    return True
```